src/nodes/html.ts

```diff
@@ -458,7 +458,7 @@ export default class HTMLElement extends Node {
 				if ((node as TextNode).isWhitespace) {
 					return;
 				}
-				node.rawText = (<TextNode>node).trimmedText;
+				node.rawText = (<TextNode>node).trimmedRawText;
```

Comment:
Because we are setting `rawText`, we want to be certain that it uses the raw version